h2o-jetty-8/build.gradle

```diff
@@ -2,9 +2,10 @@ description = "Brings Jetty 8 as the HTTP implementation used in H2O server-side
 
 dependencies {
     compile project(":h2o-webserver-iface")
-    compile 'org.eclipse.jetty:jetty-server:8.1.21.v20160908'
-    compile "org.eclipse.jetty:jetty-servlets:8.1.21.v20160908"
-    compile "org.eclipse.jetty:jetty-plus:8.1.21.v20160908"
+    ext { jetty8version = "8.2.0.v20160908" }
```

Comment:
this is usually on the top level of the build.gradle here it is quite hidden

Reply:
We might put it to properties then to have it at one place :)